MCLauncher/Themes/LightTheme.xaml.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Windows;
+
+namespace REghZyFramework.Themes
+{
+    public partial class LightTheme
+    {
+        private void CloseWindow_Event(object sender, RoutedEventArgs e)
+        {
+            if (e.Source != null)
+                try { CloseWind(Window.GetWindow((FrameworkElement)e.Source)); } catch { }
+        }
+        private void AutoMinimize_Event(object sender, RoutedEventArgs e)
+        {
+            if (e.Source != null)
+                try { MaximizeRestore(Window.GetWindow((FrameworkElement)e.Source)); } catch { }
+        }
+        private void Minimize_Event(object sender, RoutedEventArgs e)
+        {
+            if (e.Source != null)
+                try { MinimizeWind(Window.GetWindow((FrameworkElement)e.Source)); } catch { }
+        }
+
+        public void CloseWind(Window window) => window.Close();
+        public void MaximizeRestore(Window window)
+        {
+            if (window.WindowState == WindowState.Maximized)
+                window.WindowState = WindowState.Normal;
+            else if (window.WindowState == WindowState.Normal)
+                window.WindowState = WindowState.Maximized;
+        }
+        public void MinimizeWind(Window window) => window.WindowState = WindowState.Minimized;
+    }
```

Comment:
The code in here seems to be duplicated multiple times.

Reply:
I have not touched the source of this code. I believe it is like this so that you can remove any themes you don't want.

Reply:
I see. That makes sense.